standalone/index.html

```diff
@@ -218,7 +218,21 @@ <h1>WebGPU Conformance Test Suite</h1>
     <p>
       <input type="button" id="copyResultsJSON" value="Copy results as JSON">
     </p>
+    <script>
+      if ('serviceWorker' in navigator) {
```

Comment:
Need to guard this on some define like IS_STANDALONE_DEV or something

Reply:
What would that do? This file is only used for standalone dev. Should probably throw an exception if service workers aren't available.

Reply:
It's also used for the standalone build that's published to gh-pages.

Maybe we could also turn this file into a template.

Reply:
ah, right, got it.